.gitignore

```diff
@@ -1,4 +1,6 @@
 .next
 out
 node_modules
-**gin-bin*
\ No newline at end of file
+**gin-bin*
+.env
+resume-engine-cloud-sql.json
```

Comment:
Can you send me this? Or how do I get it on my own?

Reply:
Sent. I think you can redownload it from IAM > Service Accounts too.